src/utils/logging.cc

```diff
@@ -23,7 +23,11 @@
 
 #include <stdlib.h>
 #include <sys/types.h>
+#ifdef _MSC_VER
+#include <io.h>
```

Comment:
io.h is used to replace unistd.h in windows. There is no header file named unistd.h in msvc.